python/observability-online-eval/livekit/livekit-gemini.py

```
@@ -66,22 +72,33 @@ async def entrypoint(ctx: agents.JobContext):
         )
-        room = await lkapi.room.create_room(req)               # :contentReference[oaicite:0]{index=0}
+        room = await lkapi.room.create_room(
+            req
+        )  # :contentReference[oaicite:0]{index=0}
         print(f"Room created: {room}")
         session = AgentSession(
-            llm=google.beta.realtime.RealtimeModel(model="gemini-2.0-flash-exp", voice="Puck"),
-            
+            llm=google.beta.realtime.RealtimeModel(
+                model="gemini-2.0-flash-exp",
+                voice="Puck",                
```

Flake8 (7.2.0)

[warning] trailing whitespace

(W291)